pkgs/test/cc-wrapper/stdio.h

```diff
@@ -0,0 +1 @@
+static void foo() {}
```

Comment:
should this be:
```suggestion
static void foo(void) {}
```
since you don't pass it an argument?

Reply:
fixed.